docs/spec/spec.md

```diff
@@ -856,7 +856,7 @@ grpc:ClientBearerTokenAuthHandler handler = new (config);
 map<string|string[]>|grpc:ClientAuthError result = handler.enrich(requestHeaders);
 ```
 
-#### 5.1.2.7 Client - Self Signed JWT Auth
+#### 5.1.2.7 Client - self-signed JWT auth
```

Comment:
```suggestion
#### 5.1.2.7 Client - self-signed JWT auth
```
Check the industrial terms.

Reply:
https://google.aip.dev/auth/4111#:~:text=A%20self%2Dsigned%20JWT%20is,considered%20a%20standard%20authentication%20method.